airship_tempest_plugin/hacking/checks.py

```diff
@@ -190,7 +190,7 @@ def no_client_alias_in_test_cases(logical_line, filename):
     """Check that test cases don't use "self.client" to define a client.
     P103
     """
-    if "airship_tempest_plugin/tests/api" in filename:
+    if "airship_tempest_plugin/tests/api/shipyard/rbac" in filename:
```

Comment:
this is regardless of rbac. this should be enforced for all classes that instantiate a service client

Reply:
Ok, Will be modified accordingly.

Reply:
this still needs to be updated. nothing here should be hardcoded to shipyard